### DISCUSSION TITLE
Why are some batteries green while some are gray?

### DISCUSSION THREAD

**Discussion by 1920x1200** [Koenkk/zigbee2mqtt#16432] [General]
Why are some batteries green while some are gray?
Doesn't seem to be related to the amount of battery remaining. How can I make all of the colors match?

<img width="408" alt="Screen Shot 2023-01-30 at 10 11 52 PM" src="https://user-images.githubusercontent.com/38740977/215653711-3f2503e4-89fc-4454-9a50-d0e2839ef87e.png">

**Comment by sjorge** [comment]
Those that are green have a `battery_low` property, when it's set to false the icon is green.

If it does not have that property in the payload it gray. In both case it will display 5 states, 100%, 75%, 50%, 25%, 0%. When it's 25% or lower the icon blinks (IIRC it was 25%, might only blink at 0% no complete confident on that)

**Reply by 1920x1200** [reply]
Thanks for that insight! You are correct, it looks like some of my devices have the `battery low` property and some don't. Is there any way to override the ones that do to disable it so that all my batteries are the same? I really only care about the level which I can see, but having some green and some gray is distracting.

**Reply by sjorge** [reply]
You can filter out the attribute using `filtered_attributes` https://www.zigbee2mqtt.io/guide/configuration/devices-groups.html#common-device-options but i don't think this will be enough as this does not filter expose data.

Not sure if the frontend uses that data for the column though.

**Reply by 1920x1200** [reply]
I gave that a try, unfortunately it did not work.
